dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/programrule/engine/DefaultProgramRuleEngine.java

```diff
@@ -93,31 +89,31 @@ public DefaultProgramRuleEngine(
 
   @Override
   public RuleEngineEffects evaluateEnrollmentAndEvents(
-      Enrollment enrollment,
-      Set<Event> events,
-      List<TrackedEntityAttributeValue> trackedEntityAttributeValues,
-      UserDetails user) {
+      RuleEnrollment enrollment, List<RuleEvent> events, Program program, UserDetails user) {
```

Comment:
This user is the one doing the import. We are passing it here just because we decided to have it in the `TrackerBundle` and call `CurrentUserUtil` just one time.

Should we call it something like `evaluatingUser`?